src/mmrelay/plugin_loader.py

```
@@ -870,485 +900,774 @@ def _check_auto_install_enabled(config):
-                        if ref_type == "branch":
-                            logger.info(
-                                f"Cloned repository {repo_name} and checked out branch {ref_value}"
-                            )
-                        else:
-                            logger.info(
-                                f"Cloned repository {repo_name} and checked out tag {ref_value}"
-                            )
+                        logger.info(
+                            f"Cloned repository {repo_name} and checked out tag {ref_value}"
+                        )
                         return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)